libs/wingsdk/test/shared-aws/bucket.inflight.test.ts

```diff
@@ -535,3 +535,75 @@ test("tryDelete a non-existent object from the bucket", async () => {
   // THEN
   expect(objectTryDelete).toEqual(false);
 });
+
+test("Given a bucket when reaching to a non existent key, signed url it should throw an error", async () => {
+  // GIVEN
+  let error;
+  const BUCKET_NAME = "BUCKET_NAME";
+  const KEY = "KEY";
+
+  s3Mock.on(GetPublicAccessBlockCommand, { Bucket: BUCKET_NAME }).resolves({
```

Comment:
Do you really need to have`GetPublicAccessBlockCommand` and `GetBucketLocationCommand` mocks?

Reply:
@tsuf239 This is also unwanted here,  Will test and remove this one. Sorry for making this one complicated.